CMakeLists.txt

```diff
@@ -4,23 +4,24 @@
 cmake_minimum_required (VERSION 3.12)
 
 option(WARNINGS_AS_ERRORS "Treat compiler warnings as errors" ON)
-option(BUILD_CURL_TRANSPORT "Build internal http transport implementation with CURL for HTTP Pipeline" OFF)
```

Comment:
well this and all cmake files have already changed a lot.
You might want to rebase or merge master

Reply:
hank you Victor - yes, most of these CMake changes was initial work I knew I will probably throw way, but it did allow me to build at a time :)